Jenkinsfile_CNP

```diff
@@ -116,7 +116,7 @@ withPipeline(type, product, component) {
             environmentOfDependencies = "aat"
         }
         env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
-        env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+        env.ELASTIC_SEARCH_FTA_ENABLED = "false"
```

Comment:
i guess you're going to revert these ES changes before merge

Reply:
Yes - this was just to get a green build on pipeline - will revert before merge